src/pages/library/datasetBuilder/DatasetBuilderDetails.ts

```diff
@@ -0,0 +1,113 @@
+import _ from 'lodash/fp';
+import { div, h, h1, h3 } from 'react-hyperscript-helpers';
+import { ButtonOutline, spinnerOverlay } from 'src/components/common';
+import FooterWrapper from 'src/components/FooterWrapper';
+import { MarkdownViewer } from 'src/components/markdown';
+import TopBar from 'src/components/TopBar';
+import { DatasetBuilder, DatasetResponse } from 'src/libs/ajax/DatasetBuilder';
+import { useLoadedData } from 'src/libs/ajax/loaded-data/useLoadedData';
+import colors from 'src/libs/colors';
+import * as Nav from 'src/libs/nav';
+import { useOnMount } from 'src/libs/react-utils';
+import { DatasetBuilderBreadcrumbs } from 'src/pages/library/datasetBuilder/Breadcrumbs';
+
+interface DomainDisplayProps {
+  title: string;
+  displayInformation: {
+    category: string;
+    participantCount: number;
+    conceptCount: number;
+  }[];
+}
+
+const TileDisplay = (props: DomainDisplayProps) => {
+  const { title, displayInformation } = props;
+  return div([
+    h3([title]),
+    div({ style: { display: 'flex', flexWrap: 'wrap' } }, [
+      _.map(
+        (displayTile) =>
+          div(
+            {
+              style: {
+                width: '30%',
+                height: '10rem',
+                backgroundColor: 'white',
+                padding: '0.5rem 2rem',
+                marginTop: '1rem',
+                marginRight: '1rem',
+                border: `1px solid ${colors.light()}`,
+              },
+              key: _.uniqueId(''),
```

Comment:
Would `category` be better here?
```suggestion
              key: displayTile.category,
```